modules/core/shared/src/main/scala/lucuma/core/model/ElevationRange.scala

```diff
@@ -28,7 +29,9 @@ object ElevationRange {
 
   object AirMass {
     val MinValue = BigDecimal(1.0)
+    given WitnessAs[MinValue.type, BigDecimal] = WitnessAs(MinValue, MinValue)
```

Comment:
Can you expand on what does this do?

Reply:
I added this code comment.
```scala
    // manually create witnesses, since refined macro not working in scala 3
    // these are necessary to implement RefinedTypeOps
```